README.md

```diff
@@ -170,6 +170,22 @@ If you need to start fresh, you many need to sudo remove the target folder (dock
 sudo rm -rf ./target
 ```
 
+## Examples
+
+Network presets and assemblies can be combined to generate different types of nodes. Some examples:
+
+-   `$ symbol-bootstrap start -p mainnet -a dual -c customPreset.yml`
+-   `$ symbol-bootstrap start -p testnet -a peer -c customPreset.yml`
+-   `$ symbol-bootstrap start -p testnet -a demo -c customPreset.yml`
+-   `$ symbol-bootstrap start -p bootstrap -a multinode -c customPreset.yml`
+-   `$ symbol-bootstrap start -p bootstrap -a demo -c customPreset.yml`
+-   `$ symbol-bootstrap start -p bootstrap -a dual -c customPreset.yml`
+
+Although some combinations can be done, they may not be really useful. Examples that are NOT useful:
+
+-   `$ symbol-bootstrap start -p mainnet -a demo`
+-   `$ symbol-bootstrap start -p testnet -a multinode`
```

Comment:
i am not sure if we need to call these out specifically. see what @segfaultxavi thinks.

Reply:
`symbol-bootstrap start -p mainnet -a demo` could actually be useful.